index.html

```diff
@@ -361,6 +361,36 @@ <h2>A Special Time</h2>
     </div>
     </div>
 
+    <form>
+        <div>
+            <ul>
+                    <li>
+                        <label for="name">Username: </label>
+                        <input type="text" id="name" name="username">
+                    </li>
+                    <li>
+                        <label for="mail">Email: </label>
+                        <input type="email" id="mail" name="email">
+                    </li>
+                    <li>
+                        <label for="comment">What are your thoughts about this article?</label>
+                        <br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Type Here"></textarea>
```

Comment:
Add max-character limit:
comment is supposed to have max limit of 250 characters and the remaining textfield must have a max limit of 20 char